cmd/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	config "github.com/openebs/zfs-localpv/pkg/config/v1alpha1"
+	service "github.com/openebs/zfs-localpv/pkg/service/v1alpha1"
+	"github.com/openebs/zfs-localpv/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	_ = flag.CommandLine.Parse([]string{})
+	var config = config.Default()
+
+	cmd := &cobra.Command{
+		Use:   "openebs-csi-driver",
```

Comment:
This name seems very generic.
Why not `zfscsi` or something shorter
Do we see a clash with other binaries?
In other words, why should openebs be mentioned?

Reply:
yeah, will change it to zfs-driver.

Reply:
I am going to name this open-zfs-driver as this project is part of openebs.